source/StellarObject.cpp

```diff
@@ -40,6 +51,13 @@ StellarObject::StellarObject()
 
 
 
+bool StellarObject::HasSprite() const
+{
+	return usingMatches || Body::HasSprite();
```

Comment:
We should prioritize performance in the game rather than for printing data.
```suggestion
	return Body::HasSprite() || usingMatches;
```